contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -107,7 +106,9 @@ contract RyskStrategy is BaseStrategy {
     {
         IRyskLiquidityPool.DepositReceipt memory depositReceipt = ryskLqPool
             .depositReceipts(address(this));
-        uint256 currentPricePerShare = _getWithdrawalPricePerShare();
+        uint256 currentPricePerShare = ryskLqPool.withdrawalEpochPricePerShare(
```

Comment:
So the`depositReceipt` is actually the pending deposit?

Reply:
Basically yes and no in some sense. This is because of the async nature of the deposits. The deposit receipt contains information about the amount and the unredeemed shares. When the deposit method is called, only the amount filed is updated on the receipt. New shares are not minted until the pool enters a new epoch and this is when the deposit is no longer pending. However, this is not reflected in the deposit receipt until the caller interacts with the pool again, either by making another deposit or by calling `initiateWithdraw` and `redeem` functions.